.github/workflows/ci.yml

```diff
@@ -69,6 +69,27 @@ jobs:
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude .git,__pycache__,docs/source/conf.py,old,build,dist,tests/,jina/hub/
 
+  check-docstring:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - id: file_changes
+        uses: jitterbit/get-changed-files@v1
+      - name: docstring check with darlint
+        run: |
+          pip install darglint
+          pip install pydocstyle
+          for changed_file in ${{ steps.file_changes.outputs.all }}; do
```

Comment:
Why did you remove the echo? Now in the check you don't see which file it's referring to

```
Successfully installed pydocstyle-5.1.1 snowballstemmer-2.1.0
removeme.py:1 in public class `X`:
        D101: Missing docstring in public class
Error: Process completed with exit code 1.
```

This can be difficult since we have the same filenames across folders